backend/mora/service/configuration_options.py

```diff
@@ -0,0 +1,184 @@
+#
+# Copyright (c) Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import psycopg2
+import flask
+from mora import exceptions
+from .. import settings
+from .. import util
+
+
+blueprint = flask.Blueprint('configuration', __name__, static_url_path='',
+                            url_prefix='/service')
+
+
+def _get_connection():
+    conn = psycopg2.connect(user=settings.USER_SETTINGS_DB_USER,
+                            dbname=settings.USER_SETTINGS_DB_NAME,
+                            host=settings.USER_SETTINGS_DB_HOST,
+                            port=settings.USER_SETTINGS_DB_PORT,
+                            password=settings.USER_SETTINGS_DB_PASSWORD)
+    return conn
+
+
+@blueprint.route('/ou/<uuid:unitid>/configuration', methods=['POST'])
+@util.restrictargs()
+def set_org_unit_configuration(unitid):
```

Comment:
Split the body of this (along with `set_global_configuration`, as it contains more of less identical code) out into a generic `set` function - this will make implementing settings for other types trivial later on.